### PR TITLE
Document the HttpClient status code logging change

### PR DESCRIPTION
**Pull request by scottaddie**
Document the breaking change described at https://github.com/aspnet/Announcements/issues/396.

### CODE REVIEW

**Comment by scottaddie** [comment]
@gewarren @tdykstra I'm not sure what's causing the 2 build warnings. Have you seen this one before?

**Comment by tdykstra** [comment]
It's complaining about the include file not having an h1, which doesn't make sense.  I'd ignore it and log a validation bug.

**Comment by gewarren** [comment]
Usually this is a matter of excluding the includes directory in docfx.json, which it looks like is still in place: https://github.com/dotnet/docs/blob/master/docfx.json#L19

**Comment by scottaddie** [comment]
@gewarren This one is ready for your review now.

File: includes/core-changes/aspnetcore/5.0/http-httpclient-instances-log-integer-status-codes.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+The original behavior of this logging is inconsistent with other parts of ASP.NET Core that have always used integer values. The inconsistency makes logs difficult to query via structured logging systems such as [Elasticsearch](https://www.elastic.co/elasticsearch/). For more context, see [dotnet/extensions#1549](https://github.com/dotnet/extensions/issues/1549).
+
+Using integer values is more flexible than text because it allows queries on ranges of values.
+
+Adding another log value to capture the integer status code was considered. Unfortunately, doing so would introduce another inconsistency with the rest of ASP.NET Core. HttpClient logging and HTTP server/hosting logging use the same `StatusCode` key name already.
+
+#### Recommended action
+
+The best option is to update logging queries to use the integer values of status codes. This option may cause some difficulty writing queries across multiple ASP.NET Core versions right now. However, using integers for this purpose is much more flexible for querying logs.
```
Comment (gewarren) [inline includes/core-changes/aspnetcore/5.0/http-httpclient-instances-log-integer-status-codes.md:37]: Is the "right now" needed? Might be problematic for loc.

File: includes/core-changes/aspnetcore/5.0/http-httpclient-instances-log-integer-status-codes.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+    }
+    ```
+
+#### Category
+
+ASP.NET Core
+
+#### Affected APIs
+
+None
```
Comment (gewarren) [inline includes/core-changes/aspnetcore/5.0/http-httpclient-instances-log-integer-status-codes.md:76]: HttpClient?

**Review by gewarren — APPROVED** [review]
Looks good, just a couple comments/questions.

**Pull request merged**